stable/guardian/values.yaml

```diff
@@ -26,6 +26,28 @@ app:
     annotations:
       projectcontour.io/upstream-protocol.h2c: tcp
```

Comment:
Unrelated to this PR but we should avoid having this here as it is an implementation detail of our infra.
Other users could be using some other ingress controller altogether.

Reply:
Shall we remove this from the chart as part of this PR? 
WDYT @rohilsurana

Reply:
We can open a separate issue and check if everyone agrees.

Reply:
+1